.gitignore

```diff
@@ -60,6 +63,18 @@
 # build
 *build
 
+# images
+*.png
+*/*.png
+*/*/*.png
+*/*/*/*.png
+*/*/*/*/*.png
+*.pdf
+*/*.pdf
+*/*/*.pdf
+*/*/*/*.pdf
+*/*/*/*/*.pdf
```

Comment:
These lines can be reduced to just:
```.gitignore
*.png
*.pdf
```
since the .gitignore format will match these patterns in all folders.